### PR TITLE
tag: COI: Add area for automatically starting a discussion on the talkpage

### PR DESCRIPTION
**Pull request by Amorymeltzer**
[Old request](https://en.wikipedia.org/wiki/Wikipedia_talk:Twinkle/Archive_42#COI)

----

Presumably others could warrant from this, in particular {{POV}}, but likely most of the "neutrality" section.  Should we be disfavoring both of those or some subset?  Probably better to ask on-wiki.

### CODE REVIEW

File: modules/friendlytag.js [hunk 1]
```diff
@@ -1225,19 +1235,30 @@ Twinkle.tag.callbacks = {
 			pageobj.setMinorEdit(Twinkle.getPref('markTaggedPagesAsMinor'));
 			pageobj.setCreateOption('nocreate');
 			pageobj.save(function() {
-				// special functions for merge tags
+
+				// COI: Start the discussion on the talk page (mainspace only)
+				if (params.coiReason) {
+					var coiTalkPage = new Morebits.wiki.page('Talk:' + Morebits.pageNameNorm, 'Starting discussion on talk page');
+					coiTalkPage.setNewSectionText(params.coiReason + ' ~~~~');
+					coiTalkPage.setNewSectionTitle('COI tag {{subst:CURRENTMONTHNAME}} {{subst:CURRENTYEAR}}');
```
Comment (Amorymeltzer) [inline modules/friendlytag.js:1243]: Would welcome suggestions for improvement.

Comment (siddharthvp) [reply modules/friendlytag.js]: Put the date in brackets?

**Comment by Amorymeltzer** [comment]
Now updated to take into account #1216

**Comment by siddharthvp** [comment]
No need for a new function. You can just use a ternary.

**Pull request merged**